### PR TITLE
Fix tabs so content renders correctly

### PR DESCRIPTION
**Pull request by rpetrusha**
# Fix tabs so content renders correctly

## Summary

The tabbed content did not display properly, and there was some content drop-out. This PR attempts to correct it.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/core/tutorials/debugging-with-visual-studio?branch=pr-en-us-3940&tabs=visual-basic)
## Suggested Reviewers

@BillWagner @mairaw @JRAlexander

### CODE REVIEW

**Comment by mairaw** [comment]
Ah funny, I was just looking at this topic!

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 1]
```diff
@@ -28,54 +28,68 @@ You should always begin by testing your program in Debug mode. Debug mode turns
...
    ![Autos window and Immediate Window](./media/debugging-with-visual-studio/vb-autosimmediate.png)
 -->
----
 
 1. Continue program execution by selecting the **Continue** button in the toolbar or by selecting the **Debug** > **Continue** menu item. The values displayed in the console window correspond to the changes you made in the **Immediate Window**.
 
    ![Console window showing the typed value Jack at the What is your name? prompt followed by Hello Gracie on 11/1/2016 at 11:59am](./media/debugging-with-visual-studio/changed.png)
 
 1. Press any key to exit the application and end Debug mode.
+---
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]: line 89 - Gauntlet was telling me that the alt text here is too long

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Debug your C# Hello World application with Visual Studio 2017
-description: Learn how to debug a Hello World app written in C# with Visual Studio 2017.
+title: Debug your Hello World application with Visual Studio 2017
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]: I was thinking of adding some more keywords to this title to help SEO, such as
"Debug your C# or Visual Basic Hello World .NET Core application with Visual Studio 2017"

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+description: Learn how to debug a Hello World app written in C# or Visual Basic with Visual Studio 2017.
 keywords: .NET Core, .NET Core console application, .NET Core debugging
 author: BillWagner
 ms.author: wiwagn
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]: update ms.date?

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 4]
```diff
@@ -28,54 +28,68 @@ You should always begin by testing your program in Debug mode. Debug mode turns
...
    The **Immediate Window** displays the value of the string variable and the properties of the <xref:System.DateTime> value. In addition, the value of the variables is updated in the **Autos** and **Locals** windows.
 
    ![Autos window and Immediate Window](./media/debugging-with-visual-studio/autosimmediate.png)
+
+1. Continue program execution by selecting the **Continue** button in the toolbar or by selecting the **Debug** > **Continue** menu item. The values displayed in the console window correspond to the changes you made in the **Immediate Window**.
+
+   ![Console window showing the typed value Jack at the What is your name? prompt followed by Hello Gracie on 11/1/2016 at 11:59am](./media/debugging-with-visual-studio/changed.png)
+
+1. Press any key to exit the application and end Debug mode.
 # [Visual Basic](#tab/visual-basic)
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]: according to the tab spec, it should be vb not visual-basic.

**Comment by rpetrusha** [comment]
Thanks for adding the internal review URL, @JRAlexander. I forgot again!

**Comment by rpetrusha** [comment]
I've addressed your comments, @mairaw, and the build from before this commit shows the tabs building correctly.

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 5]
```diff
@@ -28,54 +28,68 @@ You should always begin by testing your program in Debug mode. Debug mode turns
...
+
+1. Program execution stops when it reaches the breakpoint and before the `Console.WriteLine` method executes. The **Autos** window displays the values of variables that are used around the current line. The **Locals** window (which you can view by clicking the **Locals** tab) displays the values of variables that are defined in the currently executing method.
+
+   ![Visual Studio application window](./media/debugging-with-visual-studio/vb-break.png)
+
+1. You can change the value of the variables to see how it affects your program. If the **Immediate Window** is not visible, display it by choosing the **Debug** > **Windows** > **Immediate** menu item. The **Immediate Window** lets you interact with the application you're debugging.
+
+1. You can interactively change the values of variables. Enter `name = "Gracie"` in the **Immediate Window** and press the Enter key.
+
 1. Enter `currentDate = new DateTime(2016,11,01,11,59,00)` in the **Immediate Window** and press the Enter key.
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]: C# and VB are not matching at this point and VB starts numbering again:
image

Seems wrong.

**Review by mairaw — CHANGES_REQUESTED** [review]
There's one part of the VB sequence that seems problematic. Other than that, everything looks great.

**Comment by rpetrusha** [comment]
@mairaw, I've fixed the VB renumbering problem.

**Comment by mairaw** [comment]
Numbering issue is fixed but you still don't have the same content on the VB side (the immediate window screenshot with some info). By design @rpetrusha?

**Comment by rpetrusha** [comment]
Yes, it's by design, @mairaw. The VB and C# Immediate windows don't behave in the same way.

**Review by mairaw — APPROVED** [review]
LGTM then.

**Pull request merged**